packages/charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -122,16 +122,18 @@ export class LineChartBase extends React.Component<
   }
 
   private _fitParentContainer(): void {
-    const { containerWidth, containerHeight } = this.state;
-    const currentContainerWidth = this.chartContainer.getBoundingClientRect().width;
-    const currentContainerHeight = this.chartContainer.getBoundingClientRect().height;
-    const shouldResize = containerWidth !== currentContainerWidth || containerHeight !== currentContainerHeight;
-    if (shouldResize) {
-      this.setState({
-        containerWidth: currentContainerWidth,
-        containerHeight: currentContainerHeight - 26
-      });
-    }
+    setTimeout(() => {
```

Comment:
As discussed offline, this might be worth revisiting in the future; however, due to the need to dynamically update the d3 axis length, this is currently an acceptable solution.